client/my-sites/earn/main.tsx

```diff
@@ -158,28 +161,25 @@ const EarningsMain = ( { section, query, path }: EarningsMainProps ) => {
 		);
 	};
 
-	const getSectionNav = ( currentSection: string ) => {
+	const getAdSectionNav = () => {
 		const currentPath = getCurrentPath();
 
 		return (
-			! currentSection.startsWith( 'payments' ) &&
-			! currentSection.startsWith( 'refer-a-friend' ) && (
```

Comment:
I deleted these checks because I'm checking the section now in the new isAdSection() method on line 81 in the new file.